fixtures/schema.js

```diff
@@ -75,6 +79,15 @@ var Channel = schema({
 
 var ChannelKey = String;
 
+var SwitchingChannel = schema({
+  '?name': [String, null], // null: use channel key
+  'trigger': ChannelKey,
+  'channels': schema({
+    '*': [Range, Array.of(1, Infinity, Range)] // '*' is the channel key of an switched channel
```

Comment:
An array of DMXRanges should at least contain two of them. Or should we just always allow arrays of ranges?